dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -3109,6 +3252,15 @@ def test_min_true_samples(self, *mocks):
         profile = dp.UnstructuredProfiler(empty_df, min_true_samples=10)
         self.assertEqual(10, profile._min_true_samples)
 
+    def test_encode(self, *mocks):
+        profiler = UnstructuredProfiler(None)
+        with self.assertRaises(NotImplementedError):
+            json.dumps(profiler, cls=ProfileEncoder)
+
+    def test_decode(self, *mocks):
+        with self.assertRaises(NotImplementedError):
+            load_profiler({"class": "UnstructuredProfiler", "data": {}})
```

Comment:
might also want a `load_from_dict` raise error that we can remove once it is functional.

Reply:
+1